verification/curator-service/ui/src/components/LinelistTable.tsx

```diff
@@ -313,17 +313,26 @@ class LinelistTable extends React.Component<Props, LinelistTableState> {
                             pageSizeOptions: [5, 10, 20, 50, 100],
                         }}
                         editable={{
```

Comment:
Does it work to have:
```ts
editable={
   this.props.user.roles.includes('curator') 
      ? {
             onRowAdd:...
             onRowUpdate:...
             onRowDelete:...
         }
       : undefined
}
```

Reply:
ah yeah, I tried but syntax failed because I forgot the undefined at the end... now it works :)